pkg/cloud/cloud.go

```diff
@@ -134,6 +134,13 @@ const (
 	fastVolumeStatusBatchMaxDelay = 500 * time.Millisecond
 )
 
+const (
+	// maxInstancesDescribed is the maximum number of instances described in each EC2 Describe Instances call
+	// TODO Question: what should a good number for this constant be? https://docs.aws.amazon.com/ec2/latest/devguide/ec2-api-pagination.html just says
```

Comment:
1000 is fine in my opinion, no reason to use less, and we'll get paginated with more